--- Makefile.orig	2013-08-05 14:21:00.000000000 -0700
+++ Makefile	2013-08-05 14:21:52.000000000 -0700
@@ -283,13 +283,6 @@
 # The DESTDIR and is added onto them to indicate where the Makefile actually
 # puts them.
 DESTDIR := $(CURDIR)/install
-PREFIX := /usr
-ifeq ($(findstring $(TARGET_OS), darwin freebsd solaris), $(TARGET_OS))
-PREFIX := /usr/local
-endif
-ifeq ($(TARGET_OS), mingw)
-PREFIX := /mingw
-endif
 prefix := $(PREFIX)
 MAN_PREFIX_EXTRA :=
 TBIN := $(DESTDIR)$(prefix)/bin
